src/components/buttons/Button.jsx

```diff
@@ -188,12 +191,14 @@ const Button = ({
       'sg-button--full-width': fullWidth,
       'sg-button--icon-only': Boolean(icon) && iconOnly,
       [`sg-button--${String(type)}-toggle-${String(toggle)}`]: toggle,
+      'sg-button--reversed-order': reversedOrder,
     },
     className
   );
 
   const iconClass = cx('sg-button__icon', {
     [`sg-button__icon--${size || ''}`]: size,
+    'sg-button__icon--right-aligned': reversedOrder,
```

Comment:
i am wondering if we could get rid of this modifier and stick to `sg-button--reversed-order` only